README.md

```diff
@@ -121,9 +121,9 @@ Edit the `scripts` section of your `package.json`:
     "deb64": "electron-installer-debian --src dist/app-linux-x64/ --dest dist/installers/ --arch amd64"
   },
   "devDependencies": {
-    "electron-installer-debian": "*",
-    "electron-packager": "*",
-    "electron-prebuilt": "*"
+    "electron-installer-debian": "^0.6.0",
+    "electron-packager": "^9.0.0",
+    "electron": "~1.0.0"
```

Comment:
Any reason why we're using v1.0.0 and not the latest? or at least the same as in the example (v1.7.0)?

Reply:
No, not particularly. It's mostly to show that you shouldn't use it verbatim.

Reply:
I'm not sure but this might create confusion and people might actually use it verbatim. Maybe we should leave the README with `*` and put a note saying that asterisks shouldn't be used literally.
We should change the `*` on the example though.
What do you think?

Reply:
Versioning is going to change as of 2.0.0 (see: https://github.com/electron/electron/pull/10336) so I'm actually inclined to change both to `^2.0.0` and also add a note saying that they should use the latest versions as necessary.

Reply:
I'm ok with that. You can put any versions you see fit and add the same note saying it shouldn't be taken verbatim and we recommend using the latest versions when possible.

Reply:
Also, should we add ourselves to the Contributor section on the README? Maybe @unindented might have a better say on this

Reply:
Personally, I'd rather just link to the GitHub contributors section of the repo.

Reply:
I'm fine either way.